Source/Charts/Components/XAxis.swift

```diff
@@ -60,6 +60,16 @@ open class XAxis: AxisBase
     /// 
     /// **default**: 1.0
     @objc open var wordWrapWidthPercent: CGFloat = 1.0
+
+    /// index of selected entry on the axis
+    ///
+    /// **default**: NSInteger.min
+    @objc open var selectedItemIndex: NSInteger = NSInteger.min
```

Comment:
Why have a default value? this should be an optional, if no item is selected then the value should be `nil` not `NSInteger.min`

Reply:
NSInteger can't be nil as I understand. Do you recommend to use NSNumber instead?

Reply:
```
    /// index of selected entry on the axis
    ///
    /// **default**: nil
    open var selectedItemIndex: NSInteger? = nil
```

Reply:
Will it work for ObjectiveC?

Reply:
I guess not, but there has been places in the charts library where they've done that.